jetty-ee9/jetty-ee9-servlet/src/main/java/org/eclipse/jetty/ee9/servlet/DefaultServlet.java

```diff
@@ -107,6 +111,16 @@
  *
  *  etags             If True, weak etags will be generated and handled.
  *
+ *  installAllowedResourceAliasChecker
```

Comment:
How about `allow-aliases` (default to false)?

Reply:
This already exists in the `ResourceServlet` used for EE10 so I don't think we should deviate from that name here, as it will be confusing when people are migrating to EE10.